easybuild/framework/easyblock.py

```diff
@@ -2892,10 +2895,15 @@ def build_and_install_one(ecdict, init_env):
             buildstats = get_build_stats(app, start_time, build_option('command_line'))
             _log.info("Build stats: %s" % buildstats)
 
-            # for reproducability we dump out the fully processed easyconfig since the contents can be affected
-            # by subtoolchain resolution (and related options) and/or hooks
-            reprod_dir = reproduce_build(app, new_log_dir)
-            _log.info("Wrote files for reproducability to %s", reprod_dir)
+            # move the reproducability files to the final log directory
+            archive_reprod_dir = os.path.join(new_log_dir, os.path.basename(reprod_dir))
+            if os.path.exists(archive_reprod_dir):
+                _log.info("Reproducability directory %s already exists, backing up" % archive_reprod_dir)
+                backup_dir = find_backup_name_candidate(archive_reprod_dir)
+                copy_dir(archive_reprod_dir, backup_dir)
+                rmtree2(archive_reprod_dir)
```

Comment:
@ocaisa You can probably use `move_file` here rather than copy & remove; despite it's name `move_file` should also work on directories.

Also, add a log entry (or move down the one above):

```python
_log.info("Existing reprod directory %s backed up to %s", archive_reprod_dir, backup_dir)